data/faq/thesis.md

```diff
@@ -0,0 +1,2 @@
+Great! This is the right place. Check out the other questions and most importantly the projects page.
```

Comment:
I'd also encourage people to add their coala related research projects here wether or not they want to do them